packages/online-shell/public/hawtconfig.json

```diff
@@ -15,5 +15,5 @@
   "login": {
     "title": "Log in with your token"
   },
-  "disabledRoutes": ["connect"]
+  "disabledRoutes": ["/connect"]
```

Comment:
I don't like the way you disable Connect. It's because it's easy that the user overrides the settings and the option `disabledRoutes` indeed is provided for the user. If disabling Connect on Hawtio Online is the specification, then we should make the logic internalised.

Looking at the old hawtio-online, it also didn't rely on `disabledRoutes` for disabling it. I wonder what was the logic to disable it.

Reply:
Created issue https://github.com/hawtio/hawtio-online/issues/200 for dealing with this separately.